apps/browser/src/_locales/en/messages.json

```
@@ -1088,6 +1088,38 @@
       },
       "description": "Shown to user after login is updated."
   },
+  "saveAsNewLoginAction": {
+    "message": "Save as new login",
+    "description": "Button text for saving login details as a new entry."
+  },
+  "updateLoginAction": {
+    "message": "Update login",
+    "description": "Button text for updating an existing login entry."
+  },
+  "saveLoginPrompt": {
+    "message": "Save login?",
+    "description": "Prompt asking the user if they want to save their login details."
+  },
+  "updateLoginPrompt": {
+    "message": "Update existing login?",
+    "description": "Prompt asking the user if they want to update an existing login entry."
+  },
+  "loginSaveSuccess": {
+    "message": "Login saved",
+    "description": "Message displayed when login details are successfully saved."
+  },
+  "loginUpdateSuccess": {
+    "message": "Login updated",
+    "description": "Message displayed when login details are successfully updated."
+  },
+  "saveFailure": {
+    "message": "Error saving",
+    "description": "Error message shown when the system fails to save login details."
+  },
+  "saveFailureDetails": {
+    "message": "Oh no! We couldn't save this. Try entering the details as a New item",
```

I know it's like this in the design, but this capitalization bugs me.

```suggestion
    "message": "Oh no! We couldn't save this. Try entering the details as a new item",
```

Relatedly, we need to figure out how to avoid concatenation here with the link (but fine as a follow-up task)

---

Ah, good call out. I will more than likely take the approach taken with the cipher name concat. I will make a ticket to address

---

https://bitwarden.atlassian.net/browse/PM-18781

---

If we want to resolve the concatenation issue now we do this:

"Try entering the details manually. **New item**"

And I think we already have a string for "new item" in the json. So that wouldn't need to be included in this one.